iiod/iiod.c

```diff
@@ -159,110 +158,109 @@ static void shutdown_avahi()
 	}
 }
 
-static void __avahi_group_cb(AvahiEntryGroup *group,
-		AvahiEntryGroupState state, void *d)
+static void __avahi_group_cb(AvahiEntryGroup *group, AvahiEntryGroupState state,
+			     void *d)
 {
 	/* Called whenever the entry group state changes */
-	if (!group)  {
+	if (!group) {
 		ERROR("__avahi_group_cb with no valid group\n");
 		return;
 	}
 
 	avahi.group = group;
 
 	switch (state) {
-		case AVAHI_ENTRY_GROUP_ESTABLISHED :
-			INFO("Avahi: Service '%s' successfully established.\n",
-					avahi.name);
-			break;
-		case AVAHI_ENTRY_GROUP_COLLISION : {
-			char *n;
-			/* A service name collision, pick a new name */
-			n = avahi_alternative_service_name(avahi.name);
-			avahi_free(avahi.name);
-			avahi.name = n;
-			INFO("Avahi: Group Service name collision, renaming service to '%s'\n",
-					avahi.name);
-			create_services(avahi_entry_group_get_client(group));
-			break;
-		}
-		case AVAHI_ENTRY_GROUP_FAILURE :
-			ERROR("Entry group failure: %s\n",
-					avahi_strerror(avahi_client_errno(
-						avahi_entry_group_get_client(group))));
-			break;
-		case AVAHI_ENTRY_GROUP_UNCOMMITED:
-			/* This is normal,
+	case AVAHI_ENTRY_GROUP_ESTABLISHED:
+		INFO("Avahi: Service '%s' successfully established.\n",
+		     avahi.name);
+		break;
+	case AVAHI_ENTRY_GROUP_COLLISION: {
+		char *n;
+		/* A service name collision, pick a new name */
+		n = avahi_alternative_service_name(avahi.name);
+		avahi_free(avahi.name);
+		avahi.name = n;
+		INFO("Avahi: Group Service name collision, renaming service to '%s'\n",
+		     avahi.name);
+		create_services(avahi_entry_group_get_client(group));
+		break;
+	}
+	case AVAHI_ENTRY_GROUP_FAILURE:
+		ERROR("Entry group failure: %s\n",
+		      avahi_strerror(avahi_client_errno(
+				      avahi_entry_group_get_client(group))));
+		break;
+	case AVAHI_ENTRY_GROUP_UNCOMMITED:
+		/* This is normal,
```

Comment:
this comment looks funky indented with the rest of the comment block

Reply:
yeah, that is weird - I tried to make it go away, but no luck so far...